targets/TARGET_ONSEMI/TARGET_NCS36510/device/TOOLCHAIN_GCC_ARM/NCS36510.ld

```diff
@@ -37,7 +39,17 @@ MEMORY {
     ENTRY(Reset_Handler)
 
     SECTIONS {
-.isr_vector :
+    .fib :
+    {
+        KEEP(*(.fib))
+    } > FIB
+
+    .trim :
+    {
+        KEEP(*(.trim))
+    } > TRIM
+
+    .isr_vector :
```

Comment:
This change, and all other changis in this file, are not related to creating an exporter. Please submit them as another PR.